src/PostEventHandler.ts

```diff
@@ -0,0 +1,53 @@
+import { SQSEvent, SQSRecord } from "aws-lambda";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { Metrics } from "@aws-lambda-powertools/metrics";
+import { Response } from "./utils/Response";
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+import { Constants } from "./utils/Constants";
+import { HttpCodesEnum } from "./models/enums/HttpCodesEnum";
+import { PostEventProcessor } from "./services/PostEventProcessor";
+
+const POWERTOOLS_METRICS_NAMESPACE = process.env.POWERTOOLS_METRICS_NAMESPACE ? process.env.POWERTOOLS_METRICS_NAMESPACE : Constants.IPR_METRICS_NAMESPACE;
+const POWERTOOLS_LOG_LEVEL = process.env.POWERTOOLS_LOG_LEVEL ? process.env.POWERTOOLS_LOG_LEVEL : Constants.DEBUG;
+const POWERTOOLS_SERVICE_NAME = process.env.POWERTOOLS_SERVICE_NAME ? process.env.POWERTOOLS_SERVICE_NAME : Constants.POSTEVENT_LOGGER_SVC_NAME;
+
+const logger = new Logger({
+	logLevel: POWERTOOLS_LOG_LEVEL,
+	serviceName: POWERTOOLS_SERVICE_NAME,
+});
+
+const metrics = new Metrics({ namespace: POWERTOOLS_METRICS_NAMESPACE, serviceName: POWERTOOLS_SERVICE_NAME });
+
+class PostEventHandler implements LambdaInterface {
+
+	@metrics.logMetrics({ throwOnEmptyMetrics: false, captureColdStartMetric: true })
+	async handler(event: SQSEvent, context: any): Promise<any> {
```

Comment:
It looks like we always return a Response type. Can we ensure that the typing wrapping the handler matches the type system AWS provide for SQS consumption, and that response also matches that?